taqueria-vscode-extension/src/lib/helpers.ts

```diff
@@ -769,6 +786,13 @@ export class VsCodeHelper {
 		});
 	}
 
+	private fixOutput(output: string): string {
+		output = output.replaceAll('', '');
```

Comment:
Is this a `noop`? It's replacing nothing with nothing

Reply:
It's actually replacing escape character. For some reason, it's not visible in Github.

<img width="413" alt="Screen Shot 2022-08-08 at 7 59 01 PM" src="https://user-images.githubusercontent.com/5188263/183554021-d5809d46-24ae-41f0-9d66-f2c6f33ef4e2.png">